scvi/module/base/_base_module.py

```diff
@@ -51,6 +56,7 @@ def __init__(
         self._kl_global = (
             kl_global if isinstance(kl_global, dict) else dict(kl_global=kl_global)
         )
+        self.record_elbo = True
```

Comment:
I think this PR will work as is, but for now, let's just remove the record elbo stuff (so always record elbo, keep defaults as you set them)